Onboarding/Tests/OnboardingTests/OnboardingTests.swift

```diff
@@ -0,0 +1,12 @@
+import XCTest
+@testable import Onboarding
+
+final class OnboardingTests: XCTestCase {
+    func testExample() throws {
+        // XCTest Documentation
+        // https://developer.apple.com/documentation/xctest
+
+        // Defining Test Cases and Test Methods
+        // https://developer.apple.com/documentation/xctest/defining_test_cases_and_test_methods
```

Comment:
Same as [this](https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579139027)